docs/iris/src/whatsnew/2.3.rst

```diff
@@ -0,0 +1,196 @@
+What's New in Iris 2.3.0
+************************
+
+:Release: 2.3.0
+:Date: 2019-10-04
+
+This document explains the new/changed features of Iris in version 2.3.0
+(:doc:`View all changes <index>`.)
+
+Iris 2.3.0 Features
+===================
+.. _showcase:
+
+.. admonition:: Increased Support for CF 1.7
+
+  Several changes have contributed to our increased support for CF 1.7:
+
+  * climatological coordinate support
+  * standard name modifier support
+  * name tokens enforcement in cell methods
+  * coordinate var_name, CF_CONVENTIONS_VERSION advanced to 1.7
+
+  You can read more about each of these below.
```

Comment:
I think this showcase needs some changing. We need to get the messaging of this right.
My proposal would be something along the lines of:

```
.. admonition:: Support for CF 1.7

  Several changes have contributed to our increased support for CF conventions
  including some CF 1.7 additions. This has included support for the
  following:

  * Climatological Coordinates
  * Standard name modifiers 
  * The Geostationary coordinate system

  More detail about these additions is outlined below.

  Furthermore, Iris now saves netCDF files with the CF conventions attribute set to 1.7.
```

Reply:
New proposal:
```
.. admonition:: Support for CF 1.7

  We have introduced several changes that contribute to Iris's support for the
  CF Conventions, including some CF 1.7 additions. We are now able to support:  

  * Climatological Coordinates
  * Standard name modifiers 
  * The Geostationary coordinate system

  For more details about these additions, see below.

  Additionally, the conventions attribute, added by Iris when saving to NetDF,
  has been updated to "CF-1.7", accordingly.
```

Reply:
👍